Motorway.cpp

```diff
@@ -0,0 +1,61 @@
+
+#include "Motorway.h"
+
+void Motorway::addCar(Car* car)
+{
+    carsArray.push_back(*car);
+}
+
+void Motorway::addMotorBike(Motorbike* motorbike)
+{
+    motorbikesArray.push_back(*motorbike);
+}
+
+void Motorway::addLorry(Lorry* lorry)
+{
+    lorriesArray.push_back(*lorry);
+}
+
+void Motorway::calculateAndPrintChanceOfAccident()
+{
+    double avgSpeedCars = 0;
+    double sumSpeedCars = 0;
+    if( carsArray.size() == 0 ) avgSpeedCars = 0;
+    else
+    {
+        for(size_t i = 0; i < carsArray.size(); ++i)
+        {
+            sumSpeedCars += carsArray[i].getSpeed();
+        }
+        avgSpeedCars = sumSpeedCars/carsArray.size();
+    }
+
+    double avgSpeedMotorbikes =  0;
+    double sumSpeedMotorbikes = 0;
+    if( motorbikesArray.size() == 0 ) avgSpeedMotorbikes = 0;
+    else
+    {
+        for(size_t i = 0; i < motorbikesArray.size(); ++i)
+        {
```

Comment:
If you're going to use `{ }` for one part of the `if/else` block, use it for both parts.
- [ ] revise